### ISSUE TITLE
Support external CUPS server

### ISSUE THREAD

**Issue by horcsinbalint** [EotvosCollegium/mars#366]
It is sometimes better to have a CUPS server that is located on a different server. (Like when the CUPS server is running nearby the printer on a different floor.) There is currently no supported method of setting the IP address and port of the CUPS server for commands `lp` and `lpstat` in mars. It could be beneficial to add an additional field in .env which allows arbitrary command line arguments to be inserted in front of the `-d` and `-W` directive respectively.

**Comment by kdmnk** [comment]
Added to #5

**Comment by horcsinbalint** [comment]
My original description did not fully show the importance of supporting external CUPS servers.

We have deployed a Raspberry PI next to the printer and connected the printer and RPI with a USB cable. The CUPS server is running on the RPI, so Mars needs to connect to that CUPS server. We have made a secure connection between the RPI and Mars's PHP FPM (with something similar to a VPN).

The changes had two main goals. One of them was to make our systems more resilient against attacks. A less important one was that from now on printers without networking capabilities would be also a great option for the dormitory.

Similarly to #364, the changes that were done in the [deployment](https://github.com/EotvosCollegium/mars/tree/deployment) branch seem to allow these settings. And I also plan to make the proper changes on the upstream [development](https://github.com/EotvosCollegium/mars/tree/development) branch. Unfortunately, I did not have enough time to do it yet.

Therefore, I would like to reopen this issue for a potential second round of evaluation.
